docs/content/direct/core-chart.md

```diff
@@ -268,7 +267,34 @@ the `kflex` CLI and one not.
     mv ~/.kube/kubeconfig-merged ~/.kube/config
     ```
 
-    Afterwards the content of a Control Plane `$cpname` can be accessed by specifing its context:
+    Afterwards the content of a KubeFlex Control Plane `$cpname` can be accessed by specifing its context:
+
+    ```shell
+    kubectl --context "$cpname" ...
+    ```
+
+3. using `import-cp-contexts.sh` script
+
+    The following covenience command can also be used to import all the KubeFlex Control Planes in the current hosting cluster as contexts of the current kubeconfig:
+
+    ```shell
+    bash <(curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/v$KUBESTELLAR_VERSION/scripts/import-cp-contexts.sh) --merge
+    ```
+
+    The script above only requires `kubectl` and `yq`.
+
+    The script accepts the following arguments:
+
+    - `--kubeconfig <filename>` specify the kubeconfig of the hosting cluster where the KubeFlex Control Planes are located. Note that this argument will override the content of the `KUBECONFIG` environment variable
+    - `--context <name>` specify a context of the current kubeconfig where to look for KubeFlex Control Planes. If this argument is not specified, then all contexts will be searched.
+    - `--names|-n <name1>,<name2>,..` comma separated list of KubeFlex Control Planes names to import. If this argument is not specified then *all* available KubeFlex Control Planes will be imported.
+    - `--replace-localhost|-r <host>` replaces server addresses "127.0.0.1" with a desired `<host>`. This parameter is useful for making KubeFlex Control Planes of type `host` accessible from outside the machine hosting the cluster.

```

Comment:
Oh crud, I just noticed that there are two different meanings of the word "host" here. Maybe say "server" instead of "host" for the first?

The flag help should explain that the replacement can either be an IPv4 (only v4 works here, right?) address or a domain name.